sdk/metricsadvisor/ai-metrics-advisor/src/metricsAdvisorAdministrationClient.ts

```diff
@@ -401,8 +417,8 @@ export class MetricsAdvisorAdministrationClient {
     if (continuationToken === undefined) {
       segmentResponse = await this.client.listDataFeeds({
         ...options.filter,
-        ...options,
-        maxpagesize: options?.maxPageSize
+        maxpagesize: options.maxPageSize,
+        skip: options.skip
```

Comment:
why changing `skip`? Also without `...options` we are losing things from `OperationOptions` like `abortSignal` etc.

Reply:
If we change to this will it be ok :
```
 ...options.filter,
        maxpagesize: options.maxPageSize,
        ...options
```
needed to change maxpagesize

Reply:
yes this should work